logging/access_log.go

```diff
@@ -139,6 +135,10 @@ func (log *AccessLog) ServeHTTP(rw http.ResponseWriter, req *http.Request, nextH
 	entry := log.logger.WithFields(logrus.Fields(fields))
 	entry.Time = startTime
 
+	if acError := acContext.Error(); acError != "" {
+		err = fmt.Errorf("error from '%s' access control: %s", acContext.Name(), acError)
```

Comment:
Just the base context, it is already an error.
```suggestion
		err = fmt.Errorf("access control: %s: %s", acContext.Name(), acError)
```